app/lib/ajax.py

```diff
@@ -0,0 +1,75 @@
+from flask import make_response
+import json
+
+
+def json_response(data, code):
+    """Return a :class:`flask.Response` with a JSON encoded object ``data`` in
+    the body.
+
+    :param dict data: The data to be put in the response body.
```

Comment:
this should in theory also work on lists, since json.dumps supports that. or at least it does with default set to json_util.default.